hedera-mirror-rest/viewmodel/customFeeViewModel.js

```diff
@@ -26,60 +27,58 @@ class CustomFeeViewModel {
    * @param {CustomFee} customFee
    */
   constructor(customFee) {
-    if (!customFee.amount && !customFee.royaltyDenominator) {
-      return;
-    }
-    this.all_collectors_are_exempt = customFee.allCollectorsAreExempt;
-    if (customFee.amountDenominator) {
-      // fractional fee
-      this.amount = {
-        numerator: customFee.amount,
-        denominator: customFee.amountDenominator,
-      };
-
-      this.denominating_token_id = EntityId.parse(customFee.tokenId).toString();
-      this.maximum = customFee.maximumAmount || undefined;
-      this.minimum = customFee.minimumAmount;
-      this.net_of_transfers = customFee.netOfTransfers;
-    } else if (customFee.royaltyDenominator) {
-      // royalty fee
-      this.amount = {
-        numerator: customFee.royaltyNumerator,
-        denominator: customFee.royaltyDenominator,
-      };
-
-      if (customFee.amount) {
-        this.fallback_fee = this._parseFixedFee(customFee);
-      }
-    } else {
-      // fixed fee
-      Object.assign(this, this._parseFixedFee(customFee));
-    }
-
-    this.collector_account_id = EntityId.parse(customFee.collectorAccountId, {isNullable: true}).toString();
+    this.fixed_fees = _.isNil(customFee.fixedFees) ? [] : customFee.fixedFees.map((f) => this._parseFixedFee(f));
```

Comment:
nit, this is shorter

```suggestion
    this.fixed_fees = customFee.fixedFees?.map((f) => this._parseFixedFee(f)) ?? [];
```

Reply:
Thanks I have simplified those lines.